packages/e26-warbler-twitter-clone/app.py

```diff
@@ -279,6 +325,35 @@ def messages_destroy(message_id):
     return redirect(f"/users/{g.user.id}")
 
 
+##############################################################################
+# Like action:
+
+@app.route('/users/add_like/<int:message_id>', methods=["POST"])
+def add_like(message_id):
+    if not g.user:
+        flash("Access unauthorized.", "danger")
+        return redirect("/")
+
+    liked_message = Message.query.get_or_404(message_id)
+    g.user.likes.append(liked_message)
+    db.session.commit()
+
+    return redirect('/')
+
+
+@app.route('/users/remove_like/<int:message_id>', methods=["POST"])
```

Comment:
`@app.route('/users/likes/<int:message_id>', methods=["DELETE"])`

Reply:
- What is the resource
- What does it belong to
- What action am I taking
- What do I send to it